sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -88,4 +91,24 @@ object IntervalUtils {
     result += MICROS_PER_MONTH * (interval.months % MONTHS_PER_YEAR)
     Decimal(result, 18, 6)
   }
+
+  def fromString(str: String): CalendarInterval = {
```

Comment:
How about keeping the method comment of `fromCaseInsensitiveString` above?

Reply:
done